src/backend/serverless/README.md

```diff
@@ -99,44 +98,46 @@ node ./deploy.js \
   -D 7200
 ```
 #### Transcoder Serverless stack
- Once you run ```deploy.js``` we will also execute deployment of Transcoder Serverless Application that contains resources for building an application that broadcast media from any URL of meeting sessions to a RTMP endpiont URL. Included is a Docker image and a serverless AWS CloudFormation template that you can deploy to your AWS account. The AWS CloudFormation template orchestrates resources (including Amazon Lambda and Amazon ECS) that run the broadcast application. When deployed, you can create Lambda test events to start/stop broadcast task in ECS. [checkout for more information or for custom deployment of transcoder app](../../../transcoding/README.md)
+Once you run ```deploy.js``` we will also execute deployment of Transcoder Serverless Application that contains resources for building an application that broadcast media from any URL of meeting sessions to a RTMP endpiont URL. Included is a Docker image and a serverless AWS CloudFormation template that you can deploy to your AWS account. The AWS CloudFormation template orchestrates resources (including Amazon Lambda and Amazon ECS) that run the broadcast application.
+> NOTE: If the backend deployment is successful there is no need to separately deploy the transcoder application but you still need to run the ECS task for broadcasting.
```

Comment:
`still need to start the broadcast by running a 'start' test event on the Transcoding Lambda and starting the MediaLive channel (specified by the main CloudFormation stack output: MediaLiveChannelId).`

Reply:
Done